src/main/java/duke/command/ByeCommand.java

```diff
@@ -0,0 +1,33 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TextUi;
+import duke.task.TaskList;
+
+/**
+ * The ByeCommand class is used when an exit command is given.
+ */
+public class ByeCommand extends Command {
+
+    /**
+     * Constructs a bye Command, which sets the boolean value isExit to true.
+     */
+    public ByeCommand() {
+        super("");
```

Comment:
Maybe you can overload the Command constructor without parameters instead of passing in an empty string.